build.gradle.kts

```diff
@@ -237,6 +237,10 @@ subprojects {
     mavenLocal()
   }
 
+  dependencies {
+        testImplementation("org.awaitility:awaitility:4.2.1")
```

Comment:
please move the dependency to here:
https://github.com/datastrato/gravitino/blob/042423429ea14896032a7eeff640b8e147cfc81a/integration-test/build.gradle.kts#L27

Reply:
BTW, it can be simplified to `testImplementation(libs.awaitility)`

Reply:
OK!